templates/resources/common/bin/run-in-container.sh.j2

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
Cool idea to merge what is common between the two `run-in-container.sh` scripts!

And maybe functions aren't so bad after all. ;-)

Reply:
Indeed, glad you see the light.